students/aminavi/session03/slicing_lab.py

```diff
@@ -0,0 +1,32 @@
+seq = [4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21]
+str = "lets do something with this"
```

Comment:
`str` is another Python built-in type name (for any Python strings!), so it's not recommended to use it as your own variable name. Basically, you won't be able to do something like `str(num)` for a number object `num` once you redefine the name `str` like this. Make sure to use different names in the future.